README.md

```diff
@@ -1,24 +1,67 @@
-# README
+## Members only (Authentication)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Project based on authenticating users on a given web page.
```

Comment:
Be more descriptive, this is the first line everybody sees when it gets into your repo